Sources/XMLCoder/Auxiliaries/XMLCoderElement.swift

```diff
@@ -107,10 +107,13 @@ struct XMLCoderElement: Equatable {
         prettyPrinted: Bool
     ) -> String {
         var string = ""
+        // Fix added for empty key adjustment in UnkeyedBox initializer
+        let nonEmptyKey = element.key != ""
+        let level = nonEmptyKey ? level + 1 : level
         string += element._toXMLString(
-            indented: level + 1, withCDATA: cdata, formatting: formatting
+            indented: level, withCDATA: cdata, formatting: formatting
         )
-        string += prettyPrinted ? "\n" : ""
+        string += (prettyPrinted && nonEmptyKey) ? "\n" : ""
         return string
```

Comment:
This is no longer preferred as far as the xmlchoicecodingkey-decode branch goes... Do you want to keep this stuff in general alive, @jsbean ?

Reply:
I think you are more knowledgable than I about this. My intuition is that we remove this change (and keep what is in the `xmlchoicekey-decode` branch).